### PR TITLE
feat: added example for the graph

### PR DESCRIPTION
**Pull request by impelcrypto**
feat: added example for the graph

### CODE REVIEW

**Review by gtg7784 — APPROVED** [review]
LGTM

**Review by wentelteefje — APPROVED** [review]

File: indexer/the-graph/.github/workflows/pr-check.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: PR Chcek
```
[the remark this answers is not in the document]
Comment (impelcrypto) [reply indexer/the-graph/.github/workflows/pr-check.yml]: Thank you. Fixed in [this](https://github.com/Soneium/soneium-examples/pull/4/commits/1147ea284a6fbd9debbf1c02f539ffd155d7fde5) commit.

File: indexer/the-graph/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+4. Run tests:
+
+   ```sh
+   yarn test
+   ```
+
+## Disclaimer
+
+We’re eager for you to start building on Soneium, but it's important that you fully understand the details of the node software and smart contracts provided here.
```
[the remark this answers is not in the document]
Comment (impelcrypto) [reply indexer/the-graph/README.md]: Fixed in [this](https://github.com/Soneium/soneium-examples/pull/4/commits/1147ea284a6fbd9debbf1c02f539ffd155d7fde5) commit.

File: indexer/the-graph/README.md
[the remark this answers is not in the document]
Comment (impelcrypto) [reply indexer/the-graph/README.md:1]: Fixed in [this](https://github.com/Soneium/soneium-examples/pull/4/commits/1147ea284a6fbd9debbf1c02f539ffd155d7fde5) commit.

File: indexer/the-graph/package.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "demo-nft-minato",
+  "license": "MIT",
+  "scripts": {
+    "codegen": "graph codegen",
+    "build": "graph build",
+    "deploy": "graph deploy --node https://api.studio.thegraph.com/deploy/ demo-nft-minato",
+    "create-local": "graph create --node http://localhost:8020/ demo-nft-minato",
+    "remove-local": "graph remove --node http://localhost:8020/ demo-nft-minato",
+    "deploy-local": "graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001 demo-nft-minato",
```
[the remark this answers is not in the document]
Comment (impelcrypto) [reply indexer/the-graph/package.json:10]: I think it’s fine, as these files and commands are created automatically by the Graph CLI, and the commands and Docker are outside the scope of the tutorial documentation.

[the remark this answers is not in the document]
Comment (Connie18) [reply indexer/the-graph/package.json:10]: oh, really. It's fine in that case. Thank you.

**Review by Connie18 — APPROVED** [review]

**Pull request merged**